docs/sources/enterprise/license/license-restrictions.md

```diff
@@ -6,66 +6,175 @@ aliases = ["/docs/grafana/latest/enterprise/license-restrictions"]
+
+When you grant Editor/Admin dashboard permissions to a Viewer, the Editor/Admin active user count increases by one as shown on the **Utilization** panel of the **Stats & licensing** page and the user’s licensed role changes to Editor/Admin.
+
+> The Editor/Admin count increases because the user can edit and save a dashboard.
+
+### Grant dashboard edit without save permissions
+
+You can grant Viewers the ability to edit but not save a dashboard. This setting is useful when you have users that want to explore dashboards, but only need the generated view temporarily and do not require an Editor/Admin license.
+
+> This setting does not affect Viewers’ license role billing.
+
+To grant dashboard edit without save permissions:
+
+1. Open the `grafana.ini` file that contains default configurations.
+
+   The name and location of the file depends on your installation. For information about locating
+   the default configurations file, refer to [Configuration](https://grafana.com/docs/grafana/latest/administration/configuration/).
+
+1. Change the `viewers-can-edit` setting to `true`.
+1. Restart Grafana Enterprise.
+
+### Determine the number of active users for each licensed role
+
+Because you can assign a user global permissions while also enhancing (or limiting) their access to specific dashboards, it might be helpful for you to periodically check license counts.
+
+To determine the number of active users for each licensed role:
+
+1. Sign in to Grafana Enterprise as a System Administrator.
+
+1. Click **Server Admin** (the shield icon).
+
+1. Click **Statistics and licensing**.
+
+1. Review the role utilization count on the **Utilization** panel.
+
+> Utilization data updates when the user signs back in to Grafana after you change their permissions.
+
+### Optimize your tiered license
+
+Grafana counts Viewer users separately from Editor/Admin users. If you reach the limit imposed by your license for either of these user types, use the following techniques to make optimum use of the licenses you purchased.
+
+**Review and update organization permissions.**
+
+You can modify organization permissions to reduce the count of one type of user. For example, if you reach the limit of Editor/Admin users, assign Viewer permissions to one (or more) Editor/Admin users who agree to read-only access to dashboards. The license count updates when a user signs back in to Grafana.
+
+**Review and update dashboard permissions.**
+
+Because Viewers can also count toward Editor/Admin usage, if they are granted access to edit specific dashboards, review your dashboard permissions and adjust accordingly. Remove Editor or Admin permissions where necessary.
+
+**Delete users.**
+
+Review the number of users you have for each type of permission and remove inactive users. Removing users immediately frees up space for new users.
```

Comment:
```suggestion
* **Review and update organization permissions.** You can modify organization permissions to reduce the count of one type of user. For example, if you reach the limit of Editor/Admin users, assign Viewer permissions to one (or more) Editor/Admin users who agree to read-only access to dashboards. The license count updates when a user signs back in to Grafana.
* **Review and update dashboard permissions.** Because Viewers can also count toward Editor/Admin usage, if they are granted access to edit specific dashboards, review your dashboard permissions and adjust accordingly. Remove Editor or Admin permissions where necessary.
* **Delete users.** Review the number of users you have for each type of permission and remove inactive users. Removing users immediately frees up space for new users.
```